Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "holeyvec"
+version = "0.1.0"
+edition = "2021"
+license-file = "LICENSE"
+description = "Vector with holes implementation. Allows removing vector's elements in a constant time."
+repository = "https://github.com/vsbogd/holeyvec"
+readme = "README.md"
+categories = [ "data-structures" ]
```

Comment:
You can use up to 5 key words and 5 category slugs.  I usually use more because it increases the changes the crate will be found.

Key words can be anything.  Personally I might add "constant-time", "stable-index", and "Vec", but you may have better ideas.

Acceptable Categories are here:  https://crates.io/category_slugs

I might add "algorithms" and "rust-patterns", but it's up to you.

Reply:
Added your suggestions in https://github.com/vsbogd/holeyvec/pull/1/commits/43b82d5a129c965207a5cbf471093fe511bdd27a